R/plot_raw_data.R

```diff
@@ -0,0 +1,86 @@
+#' Plot the raw case and death time-series data
+#'
+#' @description Produces a simple plot, using base R, of the raw case and death
+#' time-series data
+#'
+#' @param df_in A data.frame containing the outbreak data. A daily time series
+#' with dates or some other absolute indicator of time (e.g. epiday/epiweek) and
+#' the numbers of new cases and new deaths at each time point
+#'
+#' @return A plot of the two time-series on the same plot, with a legend
+#' 
+#' @export
+#'
+#' @examples
+#' # Load Ebola 1976 outbreak data
+#' data("ebola1976")
+#'
+#' plot_raw_data(ebola_1976)
+
+plot_raw_data <- function(df_in,
+                          together = TRUE) {
+  
+  dates <- df_in$date
+  cases <- df_in$cases
+  deaths <- df_in$deaths
+  
+  if(together == TRUE) {
+    # plot(dates, cases, 
+    #      col = "blue", )
+    plot(dates, cases, 
+         col = "blue", 
+         type = "s", lwd = 2,
+         xlab = "Date", ylab = "Incidence")
+    # points(dates, deaths, col = "red")
+    lines(dates, deaths, col = "red", type="s", lwd = 2,)
+    legend("topright", 
```

Comment:
Would be good to move legend (or change y-axis limits), as legend often plots over the top of the data

Reply:
Fixing this in the next PR